src/main/java/org/terasology/climateConditions/BodyTemperatureValueChangedEvent.java

```diff
@@ -0,0 +1,25 @@
+// Copyright 2020 The Terasology Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+package org.terasology.climateConditions;
+
+import org.terasology.entitySystem.event.Event;
+
+public class BodyTemperatureValueChangedEvent implements Event {
```

Comment:
As asked in another review already, maybe implementations like this can be simplified by extending a `AbstractBeforeAfterEvent<T>`? 🤔

Reply:
Yeah I think they can, So should I go ahead and add such an event? 
If yes would that system be somewhere here?: engine\src\main\java\org\terasology\entitySystem\event\
I ask because this seems like a place for very core kinda events and has only 3 events in there so trying to inquire if there's a better location for the same...